shepherd/challenge_results.py

```diff
@@ -0,0 +1,19 @@
+from collections import defaultdict
+
+results = [False, False, False, False, False, False, False, False]
```

Comment:
clean this out too, I still think something less hardcoded needs to be done in the future here. Potentially submit code via piemulator?

Reply:
Hmmm, I do like the simplicity of hardcoding, opening cross-team communication for communication sounds like work 
Maybe google sheets? Not that much better than hardcoding though

Reply:
I mean we did this thru runtime before, but that had all kind of problems too... Really just need a nice submission site thats also tied into pie. or just use google form like before, but automagic autograding with maybe instant feedback and error messages for students would be nice.

Reply:
k just deleted this file (and dependant part of shepherd.py) for now